src/applications/disability-benefits/view-payments/components/view-payments-lists/payments-returned/PaymentsReturned.jsx

```diff
@@ -1,44 +1,45 @@
-import React from 'react';
+import React, { Component } from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 import SortableTable from '@department-of-veterans-affairs/formation-react/SortableTable';
+import { clientServerErrorContent } from '../helpers';
 
-const PaymentsReceived = props => (
-  <>
-    <h3 className="vads-u-font-size--xl">Payments returned</h3>
-    <p>
-      Returned payment information is available for 6 years from the date the
-      payment was issued.
-    </p>
-    <SortableTable
-      className="va-table"
-      currentSort={{
-        value: 'String',
-        order: 'ASC',
-      }}
-      fields={props.fields}
-      data={[
-        {
-          id: 1,
-          date: 'June 4, 2020',
-          amount: '$4,321',
-          type: 'Componsation & Pension Recurring',
-          method: 'Direct Deposit',
-          bank: 'USAA FEDERAL SAVINGS BANK',
-          account: '1234567890999',
-          rowClass: 'class',
-        },
-        {
-          id: 2,
-          date: 'June 3, 2020',
-          amount: '$4,321',
-          type: 'Componsation & Pension Recurring',
-          method: 'Direct Deposit',
-          bank: 'USAA FEDERAL SAVINGS BANK',
-          account: '1234567890999',
-          rowClass: 'class',
-        },
-      ]}
-    />
-  </>
-);
+const alertClasses =
+  'vads-u-padding-y--2p5 vads-u-padding-right--4 vads-u-padding-left--2';
 
-export default PaymentsReceived;
+const PaymentsReturned = props => {
```

Comment:
This component looks almost identical to the `PaymentsReceived` component. If you wanted to, I think you could pass in props for the things that are different such as the string that gets passed to `clientServerErrorContent()` and the text for the p tag underneath the `h3`, and just use one component for both tables.

Reply:
Yep, I have another branch in the current sprint where I am refactoring these into one component that is used both places